### PR TITLE
Describe how to set qp resources

### PR DESCRIPTION
**Pull request by skonto**
## Proposed Changes 

- Adds docs for https://github.com/knative/serving/pull/14038
- Adds content for the resource-percentage annotation, previously undocumented.

### CODE REVIEW

File: docs/serving/services/configure-requests-limits-services.md [hunk 1]
```diff
@@ -23,6 +23,48 @@ spec:
...
+Alternatively, you could use a special annotation `queue.sidecar.serving.knative.dev/resource-percentage` that calculates the Queue Proxy resources as a percentage of the application's container.
+In this case there are min, max boundaries applied to the cpu and memory resource requirements:
+
+
+| Resource Requirements                               | Min     | Max     |
+|-------------------------------------------|---------|---------|
+| Cpu  Request   |  25m    |   100m  |
+| Cpu Limit |  40m    |   500m  |
+| Memory  Request   |  50Mi    |   200Mi  |
+| Memory Limit |  200Mi    |   500Mi  |
```
Comment (nak3) [inline docs/serving/services/configure-requests-limits-services.md:59]: Do we want to mention about ephemeral-storage's boundaries (I guess no boundaries?) here?

File: docs/serving/services/configure-requests-limits-services.md [hunk 2]
```diff
@@ -23,6 +23,48 @@ spec:
               cpu: 1
 ```
 
-## Additional resources
+## Configure Queue Proxy resources
+
+In order to set the Queue Proxy resource requests and limits you can either
+set them globally in the [deployment config map](../configuration/deployment.md) or you can set them at the service level using the corresponding annotations targeting cpu, memory and ephemeral-storage resource types. The previous example becomes:
```
Comment (nak3) [inline docs/serving/services/configure-requests-limits-services.md:29]:
```suggestion
set them globally in the [deployment config map](../configuration/deployment.md) or you can set them at the service level using the corresponding annotations targeting cpu, memory and ephemeral-storage resource types. The latter example becomes:
```

Sorry my English might be wrong, but "the previous" means the global setting? If so, here should be "the latter"?

File: docs/serving/services/configure-requests-limits-services.md [hunk 3]
```diff
@@ -23,6 +23,48 @@ spec:
...
+| Memory Limit |  200Mi    |   500Mi  |
+
+
+!!! note
+    If the user simultaneously sets a percentage annotation and a specific resource value via the corresponding resource annotation then the latter takes precedence.
+
+!!! warning
+    The `queue.sidecar.serving.knative.dev/resource-percentage` annotation is now deprecated and will be removed in future versions.
+
+### Additional resources
```
Comment (nak3) [inline docs/serving/services/configure-requests-limits-services.md:68]:
```suggestion
## Additional resources
```

It is better to use same head?

currently:
image

File: docs/serving/services/configure-requests-limits-services.md (hunk 1 above)
Comment (skonto) [reply docs/serving/services/configure-requests-limits-services.md:59]: There are no boundaries afaik.

File: docs/serving/services/configure-requests-limits-services.md (hunk 2 above)
Comment (skonto) [reply docs/serving/services/configure-requests-limits-services.md:29]: Yeah I am not sure either.

**Comment by skonto** [comment]
@dprotaso gentle ping pls review.

File: docs/serving/services/configure-requests-limits-services.md (hunk 1 above)
Comment (nak3) [reply docs/serving/services/configure-requests-limits-services.md:59]: So, we don't document about it?

**Comment by skonto** [comment]
@pmbanugo @snneji @dprotaso grentle ping

File: docs/serving/services/configure-requests-limits-services.md (hunk 2 above)
Comment (dprotaso) [reply docs/serving/services/configure-requests-limits-services.md:29]: maybe just say `above`

**Comment by dprotaso** [comment]
/lgtm
/approve
/hold if you want to address https://github.com/knative/docs/pull/5617#discussion_r1259804833 here 

otherwise feel free to `/unhold` and do it in a separate PR

**Comment by skonto** [comment]
@dprotaso thanks, comment addressed.

**Review by ReToCode — APPROVED** [review]
/lgtm
/approve

**Pull request merged**